docs/source/installation.rst

```diff
@@ -21,49 +21,159 @@ Requirements
 * `Requests <https://requests.readthedocs.io/>`_: used to get data from websites
 * `Pecos v0.1.9 <https://pecos.readthedocs.io/>`_: used for quality control analysis
 
-
 Install MHKiT-Python
 ^^^^^^^^^^^^^^^^^^^^^
 
-Option 1: PIP Install from Python
-""""""""""""""""""""""""""""""""""""
+Option 1: Using Conda
```

Comment:
I prefer using conda and agree with that as option 1. I think we should further update these steps and discuss git first. This was an issue we ran into at the last training course. When new users come to MHKiT, they very likely want/need the example notebooks. To get those, one needs to use git. An adjustment to these steps could proceed like this:
- Decide whether to use git. 
   - Pros: examples will be installed, enables the developer workflow, users are later one step closer to opening issues and PRs
   - Cons: requires installing / using another software that is not required to run MHKiT
- If yes git:
   - install git for windows 
   - clone repo using the single command line
   - use environment.yml file included in the repo (user workflow) OR pip install the local clone (developer workflow)
- If no git
   - curl to get environment.yml file and conda install OR pip install